snapshot-replicator/functions/remove_snapshots.py

```diff
@@ -2,6 +2,7 @@
 import datetime  
 import os
 
+source_region = os.environ['SOURCE_REGION']
```

Comment:
and you'll need to declare this env variable when triggering the lambda function no?

Reply:
yes, already did

Reply:
shouldn't it be in the `rds_snapshot_cleanup` lambda function? That only has `TARGET_REGION`